scripts/processing/05-run_ica.py

```diff
@@ -30,19 +31,28 @@ def run_ica(subject_id, tsss=None):
         raws.append(mne.io.read_raw_fif(run_fname))
     raw = mne.concatenate_raws(raws)
     if tsss:
+        # SSS reduces the data rank and the noise levels, so let's include more
+        # ICA components here
+        n_components = 0.999
         ica_name = op.join(meg_dir, subject,
                            'run_concat-tsss_%d-ica.fif' % tsss)
     else:
+        n_components = 0.98
         ica_name = op.join(meg_dir, subject, 'run_concat-ica.fif')
+    # Here we only use MEG because we only eliminate ECG artifacts, which
+    # are not prevalent in EEG (blink artifats are, but we will remove trials
```

Comment:
artifats -> artifacts

okay I stop with typos. I will try to avoid bike shedding :)